mso/resource_mso_schema_template_anp_epg.go

```diff
@@ -389,7 +574,7 @@ func resourceMSOSchemaTemplateAnpEpgRead(d *schema.ResourceData, m interface{})
 }
 
 func resourceMSOSchemaTemplateAnpEpgUpdate(d *schema.ResourceData, m interface{}) error {
-	log.Printf("[DEBUG] Template BD: Beginning Update")
+	log.Printf("[DEBUG] %s: Beginning Update", d.Id())
```

Comment:
I understand adding the Id but I would keep the Template BD part in the logs (also other ones you changed).

Reply:
ok